docs/configuration/index.md

```diff
@@ -1793,7 +1793,7 @@ client has the following configuration options.
 |`druid.broker.http.compressionCodec`|Compression codec the Broker uses to communicate with Historical and real-time processes. May be "gzip" or "identity".|`gzip`|
 |`druid.broker.http.readTimeout`|The timeout for data reads from Historical servers and real-time tasks.|`PT15M`|
 |`druid.broker.http.unusedConnectionTimeout`|The timeout for idle connections in connection pool. The connection in the pool will be closed after this timeout and a new one will be established. This timeout should be less than `druid.broker.http.readTimeout`. Set this timeout = ~90% of `druid.broker.http.readTimeout`|`PT4M`|
-|`druid.broker.http.maxQueuedBytes`|Maximum number of bytes queued per query before exerting backpressure on the channel to the data server. Similar to `druid.server.http.maxScatterGatherBytes`, except unlike that configuration, this one will trigger backpressure rather than query failure. Zero means disabled. Can be overridden by the ["maxQueuedBytes" query context parameter](../querying/query-context.md). Human-readable format is supported, see [here](human-readable-byte.md). |`0` (disabled)|
+|`druid.broker.http.maxQueuedBytes`|Maximum number of bytes queued per query before exerting backpressure on channels to the data servers. The default value is generally sufficient for small and midsize clusters. Larger clusters of hundreds of servers may need larger values (and correspondingly larger Broker heap sizes) to maximize throughput.<br /><br /> Similar to `druid.server.http.maxScatterGatherBytes`, except unlike that configuration, this one will trigger backpressure rather than query failure. Zero means disabled. Can be overridden by the ["maxQueuedBytes" query context parameter](../querying/query-context.md). Human-readable format is supported, see [here](human-readable-byte.md). |`25MB`|
```

Comment:
IMHO should we take in heap of the broker to come up with the default configuration? 
Lets say min(25MB, 5% of heap) ?
I am afraid this will break the larger clusters. 
Also lets tag it to release notes.

Reply:
Added a release notes tag.

Using a % of the heap as part of the calculation makes sense to me. It's per-query, though, so 5% seems too big. Maybe 1% or 2%?

Reply:
Sounds good to me. I just altered the calculation to be max of 25MB and 2% of heap.